StackExchange.Redis/StackExchange/Redis/Position.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+
+namespace StackExchange.Redis
+{
+    /// <summary>
+    /// A position within a stream. Defaults to <see cref="Position.New"/>.
+    /// </summary>
+    public struct Position
```

Comment:
I'm having thoughts about this name, in particular how it relates alongside `StreamPosition`. I've been thinking about how this works in some other APIs, and I wonder if the correct name here is `StreamOrigin`? I'm ... unsure, but I think that's closer and less confusing

Reply:
FWIW, I'm unsure about `StreamOrigin` (either positively or negatively), but I do think `Position` is going to conflict with enough code outside our library we should avoid a name that generic in the root namespace.

Reply:
Naming _is_ hard. If you've seen Narcos, [this is relevant/funny](https://i.redd.it/zi8clnpxxr811.jpg).

If you consider the meaning of the value, it's the point or position to begin reading a stream. So I'll just throw some names out. I'm not a huge fan of any of them but they're close. 

ReadPosition
ReadOffset 
ReadStart
InitialReadPosition (too verbose?)
StreamOffset
StreamReadOffset
StreamReadPosition

I'm not sure about including the word `Stream` in the name either, that seems to clutter the name of the pair struct that's passed to the multi-stream reading APIs (e.g., `StreamStreamOffset`, `RedisStreamStreamOffset`), those names would seem off somehow.

Hoping for a variable naming epiphany while I sleep tonight...